Add notice, when no data is loaded

cmkohnen opened the pull request:
![image](https://user-images.githubusercontent.com/59118042/217306035-f92c78bc-6e11-4f55-8127-0ee5eb04b759.png)
I'm not 100% certain with the wording / line break and definitely open for suggestions

SjoerdB93 commented:
This is great, and seems to work just as it should!
Regarding the wording, I was thinking perhaps "Press the button in the top-left corner to add data", but I'm not hard opposed to the current wording either.

cmkohnen commented:
![grafik](https://user-images.githubusercontent.com/59118042/217351923-f45bddd0-e1dd-48ed-b90e-4a6a21279931.png)
Changed the wording and added the note about files and equations for the layout's sake
